ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/management-security.j2

```diff
@@ -110,4 +110,35 @@ management security
       crl {{ crl }}
 {%         endfor %}
 {%     endfor %}
+{%     if management_security.shared_secret_profiles is arista.avd.defined %}
+{%         for profile in management_security.shared_secret_profiles | arista.avd.natural_sort("profile") %}
+   session shared-secret profile {{ profile.profile }}
+{%             if profile.secrets is arista.avd.defined %}
+{%                 for secret in profile.secrets | arista.avd.natural_sort("name") %}
+{%                     if secret.secret is arista.avd.defined %}
+{%                         set secret_cli = "secret " ~ secret.name ~ " " ~ secret.secret_type | arista.avd.default("7") ~ " " ~ secret.secret | arista.avd.hide_passwords(hide_passwords) %}
+{%                         if secret.lifetime.infinite is arista.avd.defined(true) %}
+{%                             set secret_cli = secret_cli ~ " receive-lifetime infinite transmit-lifetime infinite" %}
```

Comment:
We should not have abstraction like this in `eos_cli_config_gen`. Make sure the model represents the final running config and not the shortcuts in the CLI.

Reply:
Fixed.